.devcontainer/devcontainer.json

```diff
@@ -2,10 +2,10 @@
     // ** INSTRUCTIONS **
     // Fill in the desired values for the below elements: GITHUB_ACCOUNT and REPOSITORY_NAME
     // GITHUB_ACCOUNT is the account at which the repo is saved, e.g. "EOSPowerNetwork"
-    // REPOSITORY_NAME is the name of the actual repo, e.g. "vscode-open-eos-contract" 
+    // REPOSITORY_NAME is the name of the actual repo, e.g. "vscode-open-eos-contract"
     "build": { "args": { 
         "GITHUB_ACCOUNT": "PUT-GITHUB-ACCOUNT-HERE",
-        "REPOSITORY_NAME": "PUT-REPOSITORY-NAME-HERE"
+        "REPOSITORY_NAME": "${localWorkspaceFolderBasename}" // if you named the base folder as per README instructions you don't need to modify
```

Comment:
This change doesn't belong with the rest of the PR, and in fact I prefer not to merge this change. The user has to enter their github account here anyway, so I think it's easier to simply require the user to add their gh account and repo name, rather than restricting the naming of the directory on the host machine.

Reply:
reverted this change